e2e/install_test.go

```diff
@@ -478,9 +523,102 @@ func TestOldVersionUpgrade(t *testing.T) {
 func TestSingleNodeAirgapInstallationUbuntuJammy(t *testing.T) {
```

Comment:
should we rename this to include 'upgrade'?

Reply:
good idea. i'll rename this one